tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -37,7 +37,7 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
 
                 if (parent is { Type: SpanTypes.Sql } &&
                     HasDbType(parent, dbType) &&
-                    (parent.ResourceName == commandText || parent.GetTag(Tags.DbmDataPropagated) != null))
+                    (parent.ResourceName == commandText || commandText.StartsWith("/*dddbs=")))
```

Comment:
nit: maybe makes sense to extract this as a constant in `DatabaseMonitoringPropagator`? Not essential/blocking, just a thought